src/components/Molecules/Task/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+import CheckBox from "../../Atoms/Checkbox/index";
+import Input from "../../Atoms/Input/index";
+import EditButton from "../../Atoms/EditButton/index";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+const Task = ({
+  defalutValue,
+  defalutIsEditing,
+  onEditComplete,
+  onTaskComplete,
+  taskName,
+}) => {
+  // タスクが編集中かどうかの状態管理
+  const [isEditing, setIsEditing] = useState({ defalutIsEditing });

```

Comment:
```suggestion
  const [isEditing, setIsEditing] = useState( defalutIsEditing );
```

この代入の仕方だと下の画像のような感じになって、Booleanの値が渡せないので修正をお願いします！


![image](https://user-images.githubusercontent.com/83479175/189536628-42004f6d-8bbf-405c-96da-0a0c5f0196ea.png)